bin/switchy

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python2.7
+# vim: tabstop=4 softtabstop=4 shiftwidth=4 textwidth=80 smarttab expandtab
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import sys
+import re
+import time
+import click
+import switchy
+import logging
+
+@click.group()
+def cli():
+    pass
+
```

Comment:
You should run a linter. Pep8 requires 2 lines between blocks at the outermost indentation level:
https://www.python.org/dev/peps/pep-0008/#blank-lines